Dockerfile

```diff
@@ -1,11 +1,15 @@
-FROM python:3.8-alpine
-
+FROM python:3.8-alpine AS base
 WORKDIR /app
-
 COPY requirements.txt .
 RUN pip install --no-cache-dir -r requirements.txt
 
+FROM base AS lint
+RUN pip install --no-cache-dir pylint
 COPY *.py ./
+WORKDIR /
+CMD ["pylint", "app"]
 
+FROM base AS prod
+COPY *.py ./
```

Comment:
This is duplicated now. Did you intend that? Does pylint mutate the sources somehow?

Reply:
It's intended. Because of the `FROM base AS prod`, we're starting from the `base` image rather than the `lint` image, so we don't have the `*.py` files that we copied in there (nor `pylint` and its dependencies).